News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderDetailFragment.java

```diff
@@ -242,6 +242,16 @@ public void UpdateCurrentRssView(Context context) {
         AsyncTaskHelper.StartAsyncTask(new UpdateCurrentRssViewTask(context));
     }
 
+    /**
+     *
+     * @param context Context
+     * @param searchString The string to search for
+     */
+    public void SearchInCurrentRssView(final Context context, final String searchString) {
```

Comment:
can you please rename it to `searchInCurrentRssView` since in java method names should start with a lower case letter :)